docs/containers/tutorial-django-push-to-registry.md

```diff
@@ -0,0 +1,64 @@
+---
+Area: containers
+TOCTitle: Push Django Images to a Registry
+ContentId: 4eb2543d-84a7-4e11-b835-0d238ce7ed7a
+PageTitle: Push Django Images to a Registry in Visual Studio Code
+DateApproved: 08/12/2019
+MetaDescription: How to push a Django image to a container registry using the VS Code docker extension
+MetaSocialImage: images/tutorial/social.png
+---
+
+# Push Django Images to a Registry
+
+## Create a container registry
+
+Container registries are a secure online repository of images. With proper authentication, they enable cloud services such as Azure App Service to download the image and instantiate any number of running containers. This [cloud-native](https://azure.microsoft.com/overview/cloudnative/) best practice makes it possible to scale out a web app to handle increased loads.
+
+Registry options include the following:
+
+- The [Azure Container Registry (ACR)](https://azure.microsoft.com/services/container-registry/), a private, secure, hosted registry for your images.
+- [Docker Hub](https://hub.docker.com/), Docker's own hosted registry that provides a free way to share images.
+- A private registry running on your own server, as described on [Docker registry](https://docs.docker.com/registry/) in the Docker documentation.
+
+To create an Azure Container Registry:
+
+1. Follow [Quickstart: Create a container registry using the Azure portal](https://docs.microsoft.com/azure/container-registry/container-registry-get-started-portal), but stop before "Push image to ACR". Django deployments must include the settings described below to work properly.
```

Comment:
In docs, "below" is not used due to screenreaders where there is no "below" and "above" but "previous" and "following" are OK.
```suggestion
1. Follow [Quickstart: Create a container registry using the Azure portal](https://docs.microsoft.com/azure/container-registry/container-registry-get-started-portal), but stop before "Push image to ACR". Django deployments must include the following settings to work properly.
```